### PR TITLE
require pylint < 2.0 for py2 envs

### PR DESCRIPTION
**Pull request by vitorarrais**
## Description
pylint >= 2.x breaks when `setup.py test` runs against a py2 environment

## Solution
force `setup.py` to use pylint < 2.0 for py2 envs 

Closes issue [#1420](https://github.com/tensorflow/magenta/issues/1420)

### CODE REVIEW

**Comment by cghawthorne** [comment]
Actually, @vitorarrais, is there a way to have the pylint py2 requirement specified only in the `test_requires` portion of setup.py? We don't necessarily want to require pylint for all python 2 installations of the package.

**Comment by vitorarrais** [comment]
@cghawthorne done.

File: setup.py [hunk 1]
```diff
@@ -150,5 +150,5 @@
         'magenta': ['models/image_stylization/evaluation_images/*.jpg'],
     },
     setup_requires=['pytest-runner', 'pytest-pylint'],
-    tests_require=['pytest', 'pylint'],
+    tests_require=['pytest', 'pylint < 2.0.0;python_version=="2.7"'],
```
Comment (cghawthorne) [inline setup.py]: Does this maintain the requirement for py3? I'm not super familiar with how [environment markers](https://www.python.org/dev/peps/pep-0508/#environment-markers) work, but should it be something like:

```
'pylint < 2.0.0;python_version<"3"',
'pylint > 2.0.0;python_version>="3"',
```

Comment (vitorarrais) [reply setup.py]: I'm not familiar with this syntax either. I think you're right, even though the missing part for py3 doesn't break tests for py3 environment. This is odd. Anyway, thank you for pointing out!

**Pull request merged**